src/Controls/tests/UITests/Tests/Issues/Issue21437.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues;
+
+public class Issue21437 : _IssuesUITest
+{
+	public override string Issue => "Removing TapGestureRecognizer with at least 2 taps causes Exception";
+
+	public Issue21437(TestDevice device)
+		: base(device)
+	{ }
+
+    [Test]
+	public void ExceptionShouldNotBeThrown()
+	{
+		_ = App.WaitForElement("Item2");
+		App.DoubleClick("Item2");
+
+        //The test passes if no exception is thrown
```

Comment:
Could update a Label text double clicking and verify the updated text here; but without crash this is enough.

Reply:
Is this test enough then?